ctrd/image.go

```diff
@@ -65,6 +66,42 @@ func (c *Client) RemoveImage(ctx context.Context, ref string) error {
 	return nil
 }
 
+// SaveImage saves image to tarstream
+func (c *Client) SaveImage(ctx context.Context, exporter ctrdmetaimages.Exporter, ref string) (io.Reader, error) {
+	wrapperCli, err := c.Get(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get a containerd grpc client: %v", err)
+	}
+
+	image, err := c.GetImage(ctx, ref)
+	if err != nil {
+		return nil, err
+	}
+
+	desc := image.Target()
+	// add annotations in image description
+	if desc.Annotations == nil {
+		desc.Annotations = make(map[string]string)
+	}
+	if s, ok := desc.Annotations[ocispec.AnnotationRefName]; !ok || s == "" {
```

Comment:
how about using `exist` rather than `ok`? I think it seems to improve the readability.